tests/ui.rs

```diff
@@ -118,7 +126,10 @@ fn ui_test_usage_help_flag() {
 #[allow(unused)]
 #[macro_export]
 macro_rules! ignore {
-    ($expr:expr) => {{
-        $expr
+    ($value:expr) => {{
+        $value
+    }};
+    ($name:expr, $value:expr) => {{
+        $value
     }};
 }
```

Comment:
Frankly, I don't like this code a single bit. And why is it exactly that these tests are not run on Windows?